bin/axway.sh

```diff
@@ -1,22 +1,25 @@
 #!/bin/bash
 source $(pwd)/bin/cli.sh
 
+#Steve added for error catching
+set -o pipefail  # trace ERR through pipes
+set -o errtrace  # trace ERR through 'time command' and other functions
+#set -o nounset   ## set -u : exit the script if you try to use an uninitialised variable
+set -o errexit   ## set -e : exit the script if any statement returns a non-true return value
+
 function axway_ls {
-    sftp -q -i ~/.ssh/id_rsa_axway \
-    -o HostKeyAlgorithms=+ssh-dss \
-    -o KexAlgorithms=diffie-hellman-group14-sha1 \
+    chmod 700 ~/.ssh/id_rsa_axway
+    sftp -i ~/.ssh/id_rsa_axway \
     -o StrictHostKeyChecking=no \
-    $AXWAY_USER@$AXWAY_HOST << EOF
-    ls $@
+    $AXWAY_USER@$AXWAY_HOST || echo 'error in axway_ls' << EOF
+    ls $@ || echo 'error listing'
 EOF
 }
 register 'axway' 'ls' 'listing directory' axway_ls
 
 
 function axway_cmd {
-    sftp -q -i ~/.ssh/id_rsa_axway \
-    -o HostKeyAlgorithms=+ssh-dss \
-    -o KexAlgorithms=diffie-hellman-group14-sha1 \
+    sftp -i ~/.ssh/id_rsa_axway \
```

Comment:
Not adding -v here?

Reply:
the Verbose is very noisy and has sensitive information. It is more for debugging. We know it is connecting now so we don't need to turn on verbose. I did turn off quiet though.

Reply:
Ok.